crawl-ref/source/mon-data.h

```diff
@@ -3445,6 +3524,19 @@ DUMMY(MONS_FROG, 'F', LIGHTGREEN, "giant frog", TILEP_MONS_BULLFROG)
     {TILEP_MONS_HARPY}, TILE_CORPSE_HARPY
 },
 
+{
+    MONS_JOROGUMO, 'H', BLUE, "jorogumo",
+    M_WARM_BLOOD | M_SPEAKS | M_SEE_INVIS | M_FEMALE,
+    MR_NO_FLAGS,
+    10, MONS_JOROGUMO, MONS_JOROGUMO, MH_NATURAL | MH_EVIL, 60,
+    { {AT_HIT, AF_POISON_PARALYSE, 18}, AT_NO_ATK, AT_NO_ATK, AT_NO_ATK },
```

Comment:
Sort of funny that these have the exact same attack as hornets, but I suppose there's no overlap in range.